openfisca_core/holders.py

```diff
@@ -476,3 +474,144 @@ def __init__(self, variable_name, period, definition_period, message):
         self.period = period
         self.definition_period = definition_period
         ValueError.__init__(self, message)
+
+
+class OnDiskStorage(object):
```

Comment:
This file is getting very long. Is there any way to separate it ? Maybe the `class OnDiskStorage(object) ` could go on a separate file ?

Reply:
Would it be interesting to add a comment explaining what this class is used for?